protobuf/model_config.proto

```diff
@@ -1263,6 +1263,70 @@ message ModelSequenceBatching
     repeated Control control = 2;
   }
 
+  //@@
+  //@@  .. cpp:var:: message InitialState
+  //@@
+  //@@     Settings used to initialize data for implicit state.
+  //@@
+  message InitialState
+  {
+    //@@
+    //@@    .. cpp:var:: message StateMetaData
+    //@@
+    //@@       Meta data associated with a state.
+    //@@
+    message StateMetaData
```

Comment:
Why is this nested message needed? Can't data_type, dims and state_data be directly in InitialState?

Reply:
Yes, I would keep datatype so that the state description is complete in itself. We should verify that the state datatype matches the tensor datatype.